linetools/utils.py

```diff
@@ -28,3 +28,20 @@ def between(a, vmin, vmax):
     c = a < vmax
     c &= a >= vmin
     return c
+
+#
+def scipy_rebin(a, *args):
```

Comment:
I liked that name only as I stole it from the scipy documentation
and am somewhat expecting them to integrate it someday.

But, I can be persuaded otherwise..

X

On Jul 11, 2015, at 5:39 AM, Neil Crighton <notifications@github.com> wrote:

> In linetools/utils.py:
> 
> > @@ -28,3 +28,20 @@ def between(a, vmin, vmax):
> >      c = a < vmax
> >      c &= a >= vmin
> >      return c
> > +
> > +#
> > +def scipy_rebin(a, *args):
> Can we choose a different name? Maybe just rebin, or make_coarser_grid?
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> !DSPAM:826,55a10e87223572700532246!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/profxj/xastropy

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)